.coafile

```diff
@@ -1,7 +1,7 @@
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = **/*.rst, README.md
+files = **/*.rst
```

Comment:
As I mentioned in the other comment, this won't cover `README.rst` since there's no `/` in the path. `**.rst` instead.